R/unit.conversions.R

```diff
@@ -37,148 +37,6 @@ NULL
 #' data(unit.conversions); unit.conversions
 NULL
 
-#' Generate units conversion & parsing data
-#' 
-#' Should only need to be run once on a change in the function contents.
-#' Generates the units data to be saved in data/valid.metadata.units, 
-#' data/freeform.unit.translations, data/unit.conversions, and (all three 
-#' combined) R/sysdata.rda. These are then saved with the package.
-#' 
-#' @importFrom stats setNames
-#' @import dplyr
-#' @keywords data units internal
-generateUnitsData <- function() {
```

Comment:
oh, good point, thanks. it's private, so i'm not expecting much outcry if we just remove it, but it won't be hard to keep a deprecated version around for a while - will do!